src/main/webapp/i18n/en/exam.json

```
@@ -141,12 +142,26 @@
             "updated": "Exam updated",
             "imported": "Exam imported",
             "deleted": "Deleted exam \"{{ param }}\"",
+            "startCheckBoxText": "I confirm that I have read the rules above and that I will comply to these rules during the exam.",
```

I think "to" is the wrong preposition here
```suggestion
            "startCheckBoxText": "I confirm that I have read the rules above and that I will comply with these rules during the exam.",
```